ftdetect/elixir.vim

```diff
@@ -1,9 +1,9 @@
-au BufRead,BufNewFile *.ex,*.exs set filetype=elixir
-au BufRead,BufNewFile *.eex,*.leex set filetype=eelixir
+au BufRead,BufNewFile *.ex,*.exs setfiletype elixir
+au BufRead,BufNewFile *.eex,*.leex setfiletype eelixir
```

Comment:
This will lead to new files opened as `euphoria3` instead of `elixir`: https://github.com/elixir-editors/vim-elixir/issues/475

Reply:
Oh, I'm sorry, I should've git blamed the code a bit. I'll drop that bit. Thanks a lot for catching this! :+1: